<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_two_points">
      <attribute name="title">
       <string>Two Points</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="coord_point_1a">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Koordinat Titik 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="distance_1a">
           <property name="toolTip">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Jarak dari titik 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_titik_1a">
           <property name="text">
            <string>Titik 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="coord_point_2a">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Koordinat Titik 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="distance_2a">
           <property name="toolTip">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Jarak dari titik 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_titik_2a">
           <property name="text">
            <string>Titik 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_three_points">
      <attribute name="title">
       <string>Three Points</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="coord_point_1">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Koordinat Titik 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="distance_1">
           <property name="toolTip">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Jarak dari titik 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_titik_1">
           <property name="text">
            <string>Titik 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="coord_point_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Koordinat Titik 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="distance_2">
           <property name="placeholderText">
            <string>Jarak dari titik 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_titik_2">
           <property name="text">
            <string>Titik 2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="coord_point_3">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Koordinat Titik 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="distance_3">
           <property name="placeholderText">
            <string>Jarak dari titik 3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_titik_3">
           <property name="text">
            <string>Titik 3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
